File: number-theory/find-greatest-common-divisor-of-array.c
int findGCD(int* nums, int numsSize) {
    int a=nums[0];
    int b=nums[0];
    int c;

    for (int i=0;i<numsSize;i++){
        if (nums[i]<=a){
            a=nums[i];

        }
        if (nums[i]>=b){
        b=nums[i];
       }}
    for (int j=1;j<=b;j++){
        if (a%j==0 && b % j==0){
            c=j;
        }
    }
    return c;
}

File: greatest-common-divisor/find-greatest-common-divisor-of-array.py
class Solution(object):
    def findGCD(self, nums):
        a=0
        for i in range (0,max(nums)+1):
            if (min(nums)%(i+1)==0 and max(nums)%(i+1)==0):
                a=i+1
        return a
        """
        :type nums: List[int]
        :rtype: int
        """
        